<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChannelInfoDialog</class>
 <widget class="QDialog" name="ChannelInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>229</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>新增航道</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>名称</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>类型</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="typeComboBox">
       <item>
        <property name="text">
         <string>单向航道</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>双向航道</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>限制性航道</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>其他航道</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>码头</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>预警</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="statusComboBox">
       <item>
        <property name="text">
         <string>预警</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>非预警</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>填充色</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="colorLabel">
       <property name="minimumSize">
        <size>
         <width>23</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="colorPushButton">
       <property name="text">
        <string>选择颜色</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>限速</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="speedLineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>节</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>碰撞门限</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="collisionMaxDisLineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>米</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>偏航门限</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="yawMinDisLineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>米</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>追越门限</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="overtakLineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>米</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_12">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>抛锚门限</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dropAnchorLineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>米</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QLabel" name="label_17">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>追越最短距离</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="overtakShortDisLineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>米</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <widget class="QLabel" name="label_19">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>追越垂直距离</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="overtakVerticalDisLineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>米</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <widget class="QLabel" name="label_21">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>加速度阀值</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="acceleraThresholdLineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>米</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_16">
     <item>
      <widget class="QLabel" name="label_23">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>加速度差值</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="acceleraDfferenceLineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>米</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>逆行预警</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="converseWarnComboBox">
       <item>
        <property name="text">
         <string>开启</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>关闭</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>超载预警</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="overloadWarnComboBox">
       <item>
        <property name="text">
         <string>开启</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>关闭</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="comfirmButton">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
